package.json

```diff
@@ -37,6 +37,7 @@
     "react-native": "0.66.4",
     "react-native-camera": "^4.2.1",
     "react-native-device-info": "^9.0.2",
+    "react-native-dropdown-picker": "^5.4.2",
```

Comment:
Any reason why you chose that specific library? While its probably fine the last release is 4 months old and there are not that many downloads. This one has a lot more: https://www.npmjs.com/package/@react-native-picker/picker

Reply:
it was the best fitting library i found. the one you recommended is a picker and not a dropdown and is not fitting to the design if you check the ios version in the screenshots